canonical/juju-dashboard
# Rename Logs tab

## Done

- Rename Actions Logs tab to Logs in preparation for adding Audit logs.

## QA

- Go to a model and you should see a "Logs" tab where it used to say "Action Logs".
- Click on the Logs tab and you should see the action logs with a switcher with one item "Action Logs".

## Details

https://warthogs.atlassian.net/browse/WD-5142
